tests/TestHelpers/SetupHelper.php

```diff
@@ -993,14 +578,7 @@ public static function setSystemConfig(
 		if ($baseUrl === null) {
 			$baseUrl = self::$baseUrl;
 		}
-		return self::runOcc(
-			$args,
-			$xRequestId,
-			$adminUsername,
-			$adminPassword,
-			$baseUrl,
-			$ocPath
-		);
+		return ['code' => '', 'stdOut' => '', 'stdErr' => '' ];
 	}
```

Comment:
Same here. Or will it be a part of next PR?

Reply:
method `setSystemConfig` completely removed. :smile: